test/e2e/Tests/Fixtures/FixtureHelpers.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E
```

Comment:
tip: to reduce indentation, we should use file-scoped namespaces and drop the curly braces. Just need to add a `;` at the end of the `namespace` declaration. I would like us to do this for all new code we write.

```suggestion
namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
```

Reply:
Done